README.md

```diff
@@ -1,12 +1,13 @@
 <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
-<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1></p>
+<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1>
```

Comment:
Should close the `</p>` tag in the previous line. Also, perhaps update this heading to `Sync infrastructure data to a single place.`?

Reply:
The paragraph one line above is already closed. Do you want to nest the h1 inside the <p>.
I will adjust the heading - good point.

Reply:
> The paragraph one line above is already closed. Do you want to nest the h1 inside the `<p>`?

No, the `h1` element should not be nested within the paragraph tag. However, there is no closing tag on the above line.

Reply:
This is what I mean:
```diff
- <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
+ <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/></p>
```

Reply:
@aquamatthias The `img` tag is self-closing. The `p` tag enclosing the image is not.

Reply:
ah - overlooked it. you are right.